CHANGELOG.md

```diff
@@ -4,15 +4,17 @@
 
 -   Remove unused option `maxNumberOfProblems`;
 -   New option `ymlParsingIssueSeverityLevel` allowing to set the way parsing problems are displayed;
--   Improved project files parsing. Fewer irrelevant files are parsed. The new behavior is as follows:
-    -   parse the files mentioned in the `project.kao` file;
-    -   parse `*.yclass`, `*.yobject` and `*.ycomplete` files;
-    -   avoid parsing the files in `.generated-yml/` directory, wherever this directory stands.
 -   Improved parsing for:
     -   `forall` instructions;
     -   `(condition).check()`-like expressions;
     -   `rename` instructions.
 -   Improved location provider. `Go To Implementation` using `ctrl+F12` on a method name gives the list of the locations where the method is implemented. See [this link](https://code.visualstudio.com/docs/editor/editingevolved#_go-to-implementation) for more information.
+-   Improved project files parsing. Fewer irrelevant files are parsed. The new behavior is as follows:
+    -   parse the files mentioned in the `project.kao` file;
+    -   parse `*.yclass`, `*.yobject` and `*.ycomplete` files;
+    -   avoid parsing the files in `.generated-yml/` directory, wherever this directory stands.
+
+_Note that the whole project parsing may take some time, depending on your project size. During the project parsing, suggestions list won't be available._
```

Comment:
I thought for a while that we could add something in the ReadMe but did not find a suitable place for it. :/

Anyway:

```suggestion
_Note that parsing a whole project may take some time, depending on the project’s size. During the project parsing, suggestion lists won't be available._
```

And after that I think we’re good to go.